zingocli/src/lib.rs

```diff
@@ -11,6 +11,7 @@ use std::sync::Arc;
 use log::{error, info};
 
 use clap::{self, Arg};
+use rustls;
```

Comment:
this line should not be necessary as you are calling `rustls::crypto::ring::default_provider().install_default()` with its full path